src/Controller/InventoryController.java

```diff
@@ -6,9 +6,38 @@
 import java.util.Scanner;
 
 public class InventoryController {
-    private LinkedHashMap<String, Inventory> inventoryList;
 
+    private LinkedHashMap<String, Inventory> inventoryList;
     private static InventoryController inventoryListInstance = null;
+
+
+    private InventoryController(){
+        this.inventoryList = new LinkedHashMap<>();
+        Inventory defaultInventory = new Inventory("Inventory 1","Bodega");
+        getInventoryList().put(defaultInventory.getInventoryName(), defaultInventory);
+    }
+
+    public void setDefaultInventory(){
+        //Duda si ponerlo en el constructor o dejarlo aqui.
+        Inventory defaultInventory = new Inventory("Inventory 1","Bodega");
```

Comment:
if you have singleton the constructor will be called just once

Reply:
Done